src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -5,9 +5,28 @@
  * Do not remove no-args constructor
  */
 public class Excavator extends Machine {
+    private int operatingWeight;
+    private int bucketCapacity;
```

Comment:
Don't forget about access modifiers for your custom fields. It's important to encapsulate the fields of a class. Consider using 'private' to restrict access to them.